### ISSUE TITLE
Bump gem version for Ruby 3.0 (?)

### ISSUE THREAD

**Issue by stomar** [ruby/io-console#21]
_I guess there is a procedure I am not aware of, but just in case..._

I noticed there are a number of commits since the last gem version 0.5.6 (from 2020-02-18), both here as well as in upstream Ruby's master branch. But nominally upstream still ships with 0.5.6 (according to the gemspec there), same as 2.7.1 and 2.7.2.

I guess the gem version should be bumped and synced to upstream Ruby before the final release of 3.0, to guarantee a defined state.

_I assume this is done anyway for all default gems that have changes in upstream Ruby since their last gem release???_

**Comment by dwightwatson** [comment]
I'm not sure if this is directly related, but it looks like this may now be a blocker for Rails 7 on Ruby 3. Rails 7 now uses `debug` as a default gem instead of `byebug` and this dependency chain doesn't appear installable on Ruby 3.

```
An error occurred while installing io-console (0.5.9), and Bundler cannot
continue.
In Gemfile:
  debug was resolved to 1.3.4, which depends on
    irb was resolved to 1.3.7, which depends on
      reline was resolved to 0.2.7, which depends on
        io-console
```

**Comment by nobu** [comment]
What error while installing io-console 0.5.9?
Anyway, it doesn't seem related to this issue, could you file another one?

And this issue seems resolved since 3.0 ships 0.5.7.
